### PR TITLE
RFCP/DCCP/PYDCCP/Hadoop/SRMFNAL plugins removal

### LINKED ISSUE #7854: Remove RFIO plugins
The way I read https://github.com/cms-sw/cmssw/issues/17375#issuecomment-300428060 there is no longer any reason to support RFIO stageout . Comments, @hufnagel ?

### PR DESCRIPTION
**Pull request by amaltaro**
Fixes #7854
Fixes most of what was discussed in #7940 (still LCG implementations left).
Also added the SRMImpl to the bucket (srm command).

The subject says all plugins that were removed, including v1 (Backends) and v2 (Plugins).

I guess there will be unit tests failing and we'll have to update the TFC examples we have in WMCore as well, let's see...

### CODE REVIEW

**Comment by ticoann** [comment]
test this please

**Comment by ericvaandering** [comment]
Does this also address #7854 ?

**Comment by amaltaro** [comment]
Yes, one more issue that will vanish! Thanks!

**Comment by ericvaandering** [comment]
Do you understand the failure in the StoreResults test?

**Comment by amaltaro** [comment]
seems to be a temporary failure, just ran it on my local env and it works.

**Comment by amaltaro** [comment]
test this please

File: src/python/WMCore/Storage/Backends/__init__.py [hunk 1]
```diff
@@ -17,23 +17,13 @@
...
-from . import CPImpl
 from . import LCGImpl
-from . import HadoopImpl
-from . import UnittestImpl
+from . import XRDCPImpl
 from . import VandyImpl
-from . import TestFallbackToOldBackend
 from . import GFAL2Impl
 from . import AWSS3Impl
+from . import UnittestImpl
```
Comment (ericvaandering) [inline src/python/WMCore/Storage/Backends/__init__.py]: Can we change all these dots to the real path? Relative imports are not great.

Comment (amaltaro) [inline src/python/WMCore/Storage/Backends/__init__.py]: Aye aye Sir! I'll update it later.

**Comment by amaltaro** [comment]
@ticoann @ericvaandering please review and merge

**Comment by ericvaandering** [comment]
How about a quick "Reformat code" on a few files?

**Comment by amaltaro** [comment]
done!

**Comment by ticoann** [comment]
@amaltaro, Alan, Doesn't RFCP implementation used somewhere?

The error posted on this issue shows the code patch to RFCP.
https://github.com/dmwm/WMCore/issues/8004

**Comment by amaltaro** [comment]
According to Gaston on this comment:
https://github.com/dmwm/WMCore/issues/7940#issuecomment-316104255

rfcp* tools have been removed from the few sites that were still using it.

**Review by ticoann — APPROVED** [review]

**Pull request merged**